README.md

```diff
@@ -1 +1,95 @@
-# cars-bnb-frontend
\ No newline at end of file
+# Cars BnB App
+
+## Welcome! 👋
+> #### Cars BnB App
+> This is aone stop shop for reservation of exotic cars test driving.
+
+## Back-end App
+> [Back-end](https://github.com/Abdo9826/cars-bnb-backend)
+                             
+## Built With ✔️
+
+- React 🔥
+- redux toolkit
+- Bootstrap CSS 😋
+
+
+## Live link 🙌
+
+[LINK](https://6365ab83111cad2f422103df--stately-tulumba-fce256.netlify.app/)
+
+
+## Getting Started 🙌
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+  node js
+  npm
+
+```
+### Setup
+Clone the repository and change the directory to portfolio
+
+``` 
+  `$ git clone https://github.com/Abdo9826/cars-bnb-frontend.git`
+
+  `$ cd cars-bnb-frontend`
+
+```
+
+### Install
+Install all project dependencies by running the command below
+ 
+``` 
+ ` $ npm install`
+```
+### Usage
+
+Run
+``` 
+  `$ npm start or deploy the index.html from the build/public folder 
```

Comment:
- [ ] Your code needs a specific port to run the frontend.  This is because your frontend code requires the server to run specifically on port :3001:


![image](https://user-images.githubusercontent.com/57421823/200097371-460ce3b3-cf03-440f-9dbc-1016cfa1e49f.png)

Please, be sure to add that in your instructions:  Rails needs to run on port 3001 for the frontend to work well.

You can instruct your users on your readme instructions to run the rails server like this:

```bash
rails server -p 3001
```

That way, your users won't get confused and will be able to run your app every time 💪